src/solver/conv_asm_implicit_gemm_wrw_gtc_dynamic_xdlops.cpp

```diff
@@ -377,29 +379,21 @@ FindImplicitGemmWrwGTCDynamicXdlopsKernel(const ConvolutionContext& ctx)
                     gemm_k_global_split = if_gemm_k_global_split(
                         ctx, gemm_m_per_block, gemm_n_per_block, gemm_k_per_block);
 
-                    nxb               = 1;
-                    nxe               = 1;
                     int tunable_index = -1;
 
                     if((x * y * stride_h * stride_w == 1) && (ho * wo % 4 == 0))
                     {
-                        nxb           = 4;
-                        nxe           = 0;
+                        nxb_for_1x1   = 4;
+                        nxe_for_1x1   = 0;
```

Comment:
`nxb_for_1x1` and `nxe_for_1x1` seems have set its value in [above](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/542/files#diff-62808416bb6f57f55c44ad264c1fd1f93df6ba8766d1cd29ae7f25608ece3964R323), here seems re-assignment? And it seems `nxb_for_1x1` is always `4`?

Reply:
Here because only nxb equals 4 or 1 is generated. In 1x1 case, kernels whose nxb is 4 is prefered.

Reply:
actually in up-coming odd image size support for wrw, `nxb=4, nxe=1` is also preferred, indeed it must be used in nxe=1 and nxb >1 case.
Beside, what I mean here is that when you define this 2 symbols in line [323](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/542/files#diff-62808416bb6f57f55c44ad264c1fd1f93df6ba8766d1cd29ae7f25608ece3964R323)
![image](https://user-images.githubusercontent.com/8322173/97096838-54e9fb00-16a4-11eb-8d95-986c34341107.png) it exactly has the same value, but here it re-assign that value